src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj

```diff
@@ -18,10 +18,12 @@
 
   <ItemGroup>
     <None Remove="Testfiles\jsonFormatWindows.json" />
+    <None Remove="Testfiles\JsonLogFileFormat\issueWithoutLineAndColumn.json" />
```

Comment:
A bit strange with the different folders. (Below both with subfolders)

Reply:
The first entry is old and no longer required (files are in the sub-folder). I'll remove it